src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/ClientInterface.cs

```diff
@@ -0,0 +1,453 @@
+using System;
```

Comment:
Copyright header is likely needed here. 
Not sure if we moved away from it. I think all files have these